README.md

```diff
@@ -217,11 +217,16 @@ dataset, containing the following files:
 1. `README.md`: This file should contain a description of the dataset. See the templates for examples.
 2. `requirements.txt`: This file should be a valid `pip` specification for what is needed to install the ETL
     to build the environment. _The ETL must be runnable on Python 3.11_.
-3. `commands.yaml`: This file must have two shell command strings under the keys `build_full` and
-    `build_demo` that, if run in an environment with the requirements installed, with the specified
-    placeholder variables (indicated in python syntax, include `temp_dir` for intermediate files and
-    `output_dir` for where you want the final MEDS cohort to live) will produce the desired MEDS cohort.
+3. `dataset.yaml`: This file needs to have two keys: `metadata` and `commands`. Under `commands`, you must
+    have the keys `build_full` and `build_demo` that, if run in an environment with the requirements installed,
+    with the specified placeholder variables (indicated in python syntax, include `temp_dir` for intermediate
+    files and `output_dir` for where you want the final MEDS cohort to live) will produce the desired MEDS
+    cohort. The `metadata` key should contain information about the dataset. See the `MIMIC-IV` dataset for an
+    example of the allowed syntax here. Mandatory keys include `"description"`, `"access_policy"`, and the key
```

Comment:
Does the `access_policy` key have to correspond to one of the enum values in datasets/__init__.py? Should this be expanded on in the README file (or elsewhere?)

Reply:
It does, yes. And you're right, that should be expanded here.